### PR TITLE
add license classifier in pyproject.toml

### PR DESCRIPTION
**Pull request by equinor-ruaj**
[Recommended practice to use the classifier when using standard licenses.](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -19,7 +19,10 @@ dynamic = ["version"]
 license = { file = "LICENSE" }
 readme = { file = "README.md", content-type = "text/markdown" }
 authors = [{ name = "Equinor" }]
-classifiers = ["Programming Language :: Python"]
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
```
Comment (perolavsvendsen) [inline pyproject.toml:23]: Why not "Apache 2.0"?

**Review by perolavsvendsen — COMMENTED** [review]
Why not specific on "Apache 2.0" for the license?

File: pyproject.toml (hunk 1 above)
Comment (equinor-ruaj) [reply pyproject.toml:23]: Think it is implied that it is "2.0" and there is only one alternative on the [list of possible classifiers](https://pypi.org/classifiers/)

**Review by rwiker — APPROVED** [review]
LGTM.

**Pull request merged**